### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by samin-batra**
These are the files for the image gallery.

### CODE REVIEW

**Comment by rhnvrm** [comment]
@samin-batra Thanks for that!

BTW, I can't seem to use https://samin-batra.github.io/breeze-website/breeze_image_gallery.html

Also, it would be better if you moved the image files into the img folder.
you can use `git mv *.jpg img/`

**Comment by samin-batra** [comment]
@rhnvrm any error messages or anything? because it's opening fine from my side../
and yeah I will do that

**Comment by rhnvrm** [comment]
samin-batra github io-breeze-website-breeze_image_gallery html nexus 5x

It's also not responsive

**Comment by samin-batra** [comment]
ok, i'll try to resolve that, but also try to view it in your desktop

**Comment by rhnvrm** [comment]
Oh, I see that the arrows are not visible on the first image, you should add a shadow on them so they are visible always.

Also, you should integrate this into index.html. @FlameFractal can help you with that.

The font can also be improved. It should also have a shadow.

**Comment by rhnvrm** [comment]
samin-batra github io-breeze-website-breeze_image_gallery html iphone 5

You can use dev tools on google chrome to see how it would look on a mobile device.

**Comment by samin-batra** [comment]
ok @rhnvrm, that's helpful. Meanwhile I've fixed the responsive problem for mobile devices..

**Comment by samin-batra** [comment]
I've made all the changes to the layout, check it out..

File: breeze_image_gallery.css [hunk 1]
```diff
@@ -47,6 +47,8 @@ div.caption {
 p   {
     text-align:center;
     color:white;
+    font-size:2em;
+    text-shadow:0px 5px 25px #F4FBFE;
```
Comment (rhnvrm) [inline breeze_image_gallery.css:51]: just use #000000

**Pull request closed without merging**